raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -393,6 +394,7 @@ def set_expired_pairs(transfers_pair, block_number):
             assert pair.payee_state not in STATE_TRANSFER_PAID
             assert pair.payee_transfer.expiration < pair.payer_transfer.expiration
             pair.payee_state = 'payee_expired'
+            pair.payer_state = 'payer_expired'
```

Comment:
This change is wrong, the payer state is not expired, so it must not be set to expired. A logging message is not enough of a reason to introduce an incorrect change.

If you want to avoid the log messages, just don't emit the event if it was emitted before, and that can be checked with an `if paier.payee_state == 'payee_expired'`

Reply:
Aha. Yes Indeed. That's a much better solution. I wanted to not emit the event but was thinking of something much more convoluted.

Thanks for your tip. Will implement now.